workspaces/applications.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Applications"
+description: Learn how to access web apps running in your workspace.
+state: beta
+---
+
+You can connect to web applications installed on your workspace using the
+applications specification file located at `/coder/apps/config.yaml` of the
+workspace filesystem.
+
+![Application Launcher](../assets/workspaces/applications.png)
+
+## Enabling custom apps
+
+If you'd like to use custom apps in Coder, you can enable this feature in the
+UI:
+
+1. In the top-right, click on your avatar and select **Feature Preview**
+1. Click **Generic applications** and select **Enable**.
+
+## Application specification file
+
+The application specification file allows you to define what Coder needs to
+launch the application. Within the file, you can specify the following for each
+application:
+
+```yaml
+# /coder/apps/config.yaml
+
+apps:
+  # Name of application in launcher
+  - name: projector
+    # Application scheme - must be http or https
+    scheme: http
+    # Application port
+    port: 9999
+    # Working directory
+    dir: /home/coder
+    # File path to icon used in application launcher
+    icon_path: /home/coder/goland.svg
+    # Command to start the application
+    command: /home/coder/.local/bin/projector
+    # Array of arguments for command
+    args: ["run"]
+    # Health checks to get running application status
+    # Can use exec or http health checks to localhost
+    health_check:
+      exec:
+        command: "pgrep"
+        args: ["projector"]
+      http:
+        scheme: "http"
+        path: "/"
+        port: 9999
+```
+
+**Notes**:
+
+- All top-level fields in the `config.yaml` file are required
+- You must include at least one health check, though we recommend both. The
```

Comment:
```suggestion
- You must include at least one health check. The
```

We actually only check one on the backend. If the http health check is there, we use that and only that (even if the exec health check is there).